### DISCUSSION TITLE
AV1 grain synthesis with NVencC Encoder (or perhaps I don't know how to do it with the current version)

### DISCUSSION THREAD

**Discussion by cleverestx** [cdgriffith/FastFlix#546] [Ideas]
With Handbrake you can apparently do:  _film-grain-denoise=0_  which is recommended to preserve details lost without the denoise part...

I see in HEVC encoding there is a FFMPEG custom line that apparently it's possible to so some sort of command like this if I'm not mistaken, but with AV1, there appears to be no such custom option. Can this be added or can someone tell me how to do this if it's already there? Thank you very much.

**Comment by cleverestx** [comment]
Am I missing something obvious? How am I the only person interested besides two upvotes to my request? Hello? Anyone? Is this a hopeless request?

**Comment by cdgriffith** [comment]
Grain removal is sadly not part of NVencC https://github.com/rigaya/NVEnc/issues/433
